frontend/src/app/admin/applications/widgets/application-review-modal.widget.ts

```diff
@@ -0,0 +1,51 @@
+import { Component, Inject, OnDestroy } from '@angular/core';
+import { MAT_DIALOG_DATA } from '@angular/material/dialog';
+import { HttpClient } from '@angular/common/http';
+import { Application } from '../admin-application.model';
+import { MatSnackBar } from '@angular/material/snack-bar';
```

Comment:
This also does not seem to be implemented, am I correct? I could not find this in the admin page.